hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1875,16 +1875,17 @@
     // ENTROPY_SRC (pre-verified IP) integration tests:
     {
       name: chip_sw_entropy_src_ast_rng_req
+      //TODO: A dv enginner should add an asertion to check the connectivity for this test.
       desc: '''Verify the RNG req to ast.
 
             - Program the entropy src in normal RNG mode.
             - Route the entropy data received from RNG to the FIFO.
             - Verify that the FIFO depth is non-zero via SW - indicating the reception of data over
-              the AST RNG interface.
+              the AST RNG interface. 
```

Comment:
Please remove the extra space at the end of this line.

Reply:
Done